fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -3383,6 +3383,9 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static String default_mv_partition_refresh_strategy = "strict";
 
+    @ConfField(mutable = true)
+    public static String default_mv_refresh_mode = "pct";
```

Comment:
why not use "auto" for default refresh mode? then introduce a decision-maker to determine which real refresh mode takes effect.

Reply:
Good question.

I want to use `auto` for more intelligent strategy for automatic switches between `pct` and `increment` later.

And for current's implementions, we are using `pct` (partition change tracking) by default.